### PR TITLE
add coulomb to 2.13 community build

### PR DESCRIPTION
**Pull request by erikerlandson**
My primary motivation is that https://github.com/scala/scala/pull/6050 introduced improvements to implicit chaining that coulomb depends on, and coulomb's implicit rules are a stress-test for these facets of chaining.

### CODE REVIEW

**Comment by erikerlandson** [comment]
I'll have to tweak this to add some `%Provided` deps

**Comment by erikerlandson** [comment]
`deps.inject` and `extra.commands` aren't working at all for this. I can get it to stop failing with "library xxx not provided" but then the compilation still doesn't see the deps and it's failing on scala `import`s

**Comment by SethTisue** [comment]
I added a couple necessary tweaks. let's see what Jenkins currently says, as a baseline: ~~https://scala-ci.typesafe.com/job/scala-2.13.x-jdk11-integrate-community-build/1352/~~ ~~https://scala-ci.typesafe.com/job/scala-2.13.x-jdk11-integrate-community-build/1353/~~ ~~https://scala-ci.typesafe.com/job/scala-2.13.x-jdk11-integrate-community-build/1354/~~
https://scala-ci.typesafe.com/job/scala-2.13.x-jdk11-integrate-community-build/1355/

**Comment by erikerlandson** [comment]
So, I had deleted all of the other project includes "to save time" when I ran run.sh - now suspecting I shot myself in the foot?

**Comment by SethTisue** [comment]
> So, I had deleted all of the other project includes "to save time" when I ran run.sh - now suspecting I shot myself in the foot?

Right. `./narrow coulomb` is the way to limit it to just the projects that are needed, but the `narrow` script depends on having an accurate entry in `dependencies.txt`.

**Comment by erikerlandson** [comment]
Thanks @SethTisue !

**Comment by SethTisue** [comment]
thank you for the contribution! bab8a5ced98e3abad5891df0edfab7a57267c448 (force-pushed) restores your authorship on the commit, not sure how that went wrong the first time

File: dependencies.txt [hunk 1]
```diff
@@ -211,4 +211,5 @@ github4s: base64, better-monadic-for, blaze, boopickle, cats, cats-effect, cats-
...
+coulomb: akka, algebra, better-monadic-for, blaze, boopickle, cats, cats-effect, cats-effect-testing, cats-testkit-scalatest, circe, claimant, cloc-plugin, discipline, discipline-scalatest, discipline-specs2, enumeratum, fansi, fastparse, fastparse-scalameta, fs2, http4s, http4s-parboiled2, jawn, jawn-fs2, json4s, kind-projector, log4cats, log4s, machinist, magnolia, mdoc, mercator, metaconfig, minitest, munit, paiges, play-json, portable-scala-reflect, pprint, pureconfig, pureconfig-more, refined, scala, scala-collection-compat, scala-java8-compat, scala-js-stubs, scala-parser-combinators, scala-xml, scalacheck, scalacheck-shapeless, scalameta, scalapb, scalatags, scalatest, scalatest-3-0, scalatestplus-junit, scalatestplus-scalacheck, scodec-bits, sconfig, shapeless, silencer, simulacrum, singleton-ops, sourcecode, specs2, specs2-more, spire, spray-json, tut, twirl, unique, utest, vault, wartremover
```
Comment (SethTisue) [inline dependencies.txt:214]: @erikerlandson I suspect that after https://github.com/erikerlandson/coulomb/pull/75, this list wouldn't be so long

**Pull request merged**